src/main/java/domain/board/ChessBoardFactory.java

```diff
@@ -41,4 +43,12 @@ public static ChessBoard createInitialChessBoard() {
         }
         return new ChessBoard(pieceMap);
     }
+
+    public static ChessBoard loadPreviousChessBoard(List<PieceDto> pieceDtos, final Color turn) {
+        return pieceDtos.stream()
+                .collect(Collectors.collectingAndThen(Collectors.toMap(
+                        PieceDto::getPosition,
+                        PieceDto::getPiece
+                ), board -> new ChessBoard(board, turn)));
```

Comment:
```suggestion
        return pieceDtos.stream()
                .collect(
                    Collectors.collectingAndThen(
                        Collectors.toMap(PieceDto::getPosition, PieceDto::getPiece),
                        board -> new ChessBoard(board, turn)
                    )
                );
```
괄호 깊이에 맞게 줄바꿈을 해주세요!

Reply:
이건 컨벤션의 영역이라고만 생각했어요. 하지만 지난 리뷰어 분에게도 거의 동일한 리뷰를 받았고, 이번에 피케이도 말씀해 주시네요! 이제 확실히 지키겠습니다.